app/views/dashboard/autoyast.xml.erb

```diff
@@ -10,6 +10,7 @@
         <source><![CDATA[
         #!/bin/sh
         echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "grains_refresh_every: 10" >> /etc/salt/minion.d/master.conf
```

Comment:
Not sure if it supports a `SIGHUP`-like signal that only reloads the config...

Approving though.